consent/types.go

```diff
@@ -367,6 +367,9 @@ type ConsentRequest struct {
 	// Context contains arbitrary information set by the login endpoint or is empty if not set.
 	Context map[string]interface{} `json:"context,omitempty"`
 
+	// HandledAt contains the timestamp the consent request was handled.
+	HandledAt time.Time `json:"handled_at,omitempty"`
```

Comment:
This value still needs to be set somewhere, otherwise it will always be nil.

Reply:
Yes, we missed this and added this to the PR.